Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -1,192 +1,70 @@
-  background-size: 1px 20px;
-  background-color: #C5E0FC;
+  display: flex;
+  justify-content: space-evenly;
+  box-sizing: border-box;
+  padding: var(--remotion-themed-spacing-half)
 }
 
-*.tabStripTabSelected a
+.tabStripTabWrapper .tabStripTab
 {
-  border: solid 1px #AAAAAA;
-  background-image: none;
-  background-color: White;
-  color: Black;
-  text-decoration: none;
+  border: var(--remotion-themed-border);
+  border-radius: var(--remotion-themed-border-radius);
 }
 
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie *.tabStripTab a,
-body.msie *.tabStripTabSelected a
+.tabStripTab:hover
 {
-  position: relative;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie *.tabStripTab a::before,
-body.msie *.tabStripTabSelected a::before
-{
-  content: "";
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
-}
-
-*.tabStripTab.disabled a,
-*.tabStripTab.disabled a:focus,
-*.tabStripTab.disabled a:hover
-{
-  color: Gray;
-  background-color: #EFEFF0;
-}
-
-*.tabStripTab.disabled > img
-{
-  opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
-          progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
-}
-
-/* Content area of the tab */
-*.tabStripTabsPane *.anchorBody
-{
-  padding-top: 0.2em; /* Value must also be set in the Non-IE browsers section as padding for the separator. */
-  padding-bottom: 0.2em; /* Value must also be set in the Non-IE browsers section as padding for the separator. */
-  padding-left: 0.2em;
-  padding-right: 0.2em;
-  white-space: nowrap;
-  display: block;
-}
-
-*.tabStripTabsPane *.anchorBody > img
-{
-  vertical-align: middle;
-}
-
-*.tabStripTabsPane *.anchorBody > span
-{
-}
-
-
-*.tabStripTabsPane ul *.tabStripTabWrapper
-{
-  float: left;
-  white-space: nowrap;
-}
-
-*.tabStripTabSeparator
-{
-  float: left;
-  width: 0.3em; /* Value must also be set in the Non-IE browsers section as left-margin of the tab. */
-  margin: 0;
-  display: inline-block;
-}
-
-*.tabStripTab,
-*.tabStripTabSelected
-{
-  position: relative;
-  display: inline-block;
-}
-
-*.tabStripTabSeparator span,
-*.tabStripTabsPane a
-{
-  display: block;
-}
-
-*.tabStripTabsPane *.last
-{
-  display: block;
-  clear: left;
+  border: var(--remotion-themed-border-highlight);
+  background-color: var(--color-highlight);
+  cursor: pointer;
 }
 
-*.tabStripTabSeparator span
+.tabStripTabSelected
 {
-  padding-top: 1px; /* The padding used for the content span. */
-  padding-bottom: 1px; /* The padding used for the content span. */
+  border: var(--remotion-themed-border-highlight);
+  background-color: var(--color-highlight);
+  border-radius: var(--remotion-themed-border-radius);
+  cursor: pointer;
 }
 
-*.tabStrip div.clearingPane
+.tabStripTab.disabled a,
+.tabStripTab.disabled a:focus,
+.tabStripTab.disabled a:hover
```

Comment:
we use not(.disabled) for focus and hover so we don't have to repair the style here but instead just set the color for .disabled and done.